eodag/utils/logging.py

```diff
@@ -18,18 +18,27 @@
 
 import logging.config
 
+disable_tqdm = False
 
-def setup_logging(verbose):
+
+def setup_logging(verbose, no_progress_bar=False):
     """Define logging level
 
     :param verbose: Accepted values:
 
-                    * 0: no logging
-                    * 1 or 2: INFO level
+                    * 0: no logging with mutted progress bars
+                    * 1: no logging but still displays progress bars
+                    * 2: INFO level
                     * 3: DEBUG level
     :type verbose: int
+
+    :param verbose: (optional) disable progress bars (default=False)
+    :type verbose: bool
     """
-    if verbose == 0:
```

Comment:
I'm not sure I get it, what's the difference between setup_logging(0) and setup_logging(1)?